catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -123,6 +128,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
 		return super.onOptionsItemSelected(item);
 	}
 
+	public boolean isRecoveredFromCrash() {
+		return PreferenceManager.getDefaultSharedPreferences(this).getBoolean(RECOVERED_FROM_CRASH, false);
+	}
+
+	public boolean checkIfCrashRecoveryAndFinishActivity(final Activity context) {
```

Comment:
It was not my idea to put context as final. I think is was same as before or I think IDE did the trick there. 😅